### PR TITLE
[MM-30510] Fix sorted store and api times in output Markdown

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

PR fixes some rather stupid bugs introduced in https://github.com/mattermost/mattermost-load-test-ng/pull/390.
Funny thing, the attached sample report was full of issues but nobody noticed (myself included :facepalm: ).

#### Ticket

https://mattermost.atlassian.net/browse/MM-30510

### CODE REVIEW

**Review by agnivade — COMMENTED** [review]
Lol, oh dear. Wondering if we can add a test for this?

**Comment by streamer45** [comment]
> Lol, oh dear. Wondering if we can add a test for this?

We could but it would probably need some re-work because it was the markdown to be bugged. Sorting was fine, just printing the wrong data. I'll see if I can come up with something.

**Comment by streamer45** [comment]
Clearly not the most enjoyable of test I wrote but it should do the job.

**Review by agnivade — APPROVED** [review]
It's a compromise. :) I think given the general bugs we've had around this one, good to have some test coverage in this area.

File: loadtest/report/output.go [hunk 1]
```diff
@@ -48,16 +48,26 @@ func printSummary(c comp, target *os.File, base Report, cols int) {
...
 			}
-			fmt.Fprintf(target, "| %s", label)
-			fmt.Fprintf(target, " |  %s", metric)
-			fmt.Fprintf(target, "| %s", getDuration(float64(base.AvgStoreTimes[label])))
-			fmt.Fprintf(target, "| %s | %s | %.3f", d[0].actual, d[0].delta, d[0].deltaPercent)
+			// Only show requested data.
+			if showImproved && d[0].delta > 0 || !showImproved && d[0].delta < 0 {
+				break
+			}
+			// Skip deltas smaller than 2ms.
```
Comment (isacikgoz) [inline loadtest/report/output.go:63]: Should we give an info about this to the end of the output? something like "deltas below 2ms are discarded".

File: loadtest/report/output.go [hunk 2]
```diff
@@ -48,16 +48,26 @@ func printSummary(c comp, target *os.File, base Report, cols int) {
 			} else if metric == "p99" {
 				d = measurement[1]
 			}
+			// We only handle comparisons between 2 reports.
 			if len(d) != 1 {
 				break
 			}
-			if (showImproved && d[0].deltaPercent >= 0) || (!showImproved && d[0].deltaPercent <= 0) {
+			// Skip delta percentages smaller than 1.
```
Comment (isacikgoz) [inline loadtest/report/output.go:55]: Same.

**Review by isacikgoz — APPROVED** [review]
Next level testing!! 🚀 I have a non-blocking question/suggestion.

File: loadtest/report/output.go (hunk 1 above)
Comment (streamer45) [reply loadtest/report/output.go:63]: We could document this at https://github.com/mattermost/mattermost-load-test-ng/blob/master/docs/compare.md.

**Pull request merged**